tests/test_s3.py

```diff
@@ -62,4 +62,9 @@ async def _test():
             bucket_data = await list_files(client, s3_bucket, "foo")
             assert sorted([x["Key"] for x in bucket_data]) == ["foo/test.csv"]
 
+            await upload_file(client, s3_bucket, static_dir / "test.csv", "foo/test.tsv", acl="public-read")
+            await delete_files(client, s3_bucket, ["foo/test.tsv", "foo/test.csv"])
```

Comment:
can you add a check to verify the return type ?

Reply:
I ve add 
```python
    # Check for errors in the response
    if "Errors" in response and response["Errors"]:
        return False
    return True
````
in the end of the function

and test it
with 
```python
            response = await delete_files(client, s3_bucket, ["foo/test.tsv", "foo/test.csv"])
            assert response is True
````